CRM/Contribute/Form/Contribution.php

```diff
@@ -618,6 +618,29 @@ public function buildQuickForm() {
         TRUE, array('onChange' => "return showHideByValue('payment_instrument_id','4','checkNumber','table-row','select',false);")
       );
     }
+    
+    if ($this->_context == 'standalone') {
```

Comment:
This seems to be adding recurring options for standalone - but aren't they already there for non standalone? Which implies there is an IF excluding standalone somewhere? Which could be un-iffed rather than adding new ifs?